readme.md

```diff
@@ -93,6 +93,8 @@ Full schematics:
 
 Part list:
 
+You can easily buy the parts listed in the [1-click-bom](1-click-bom.tsv) file using the [1-click BOM extension](https://1clickbom.com). 
```

Comment:
Could you make this take less vertical space by moving the link to the part list line? I.e.
```
Part list (add to cart with [1-click BOM](https://1clickbom.com)):
```